src/main/java/Board.java

```diff
@@ -9,9 +12,74 @@ public Grid getGrid() {
         return grid;
     }
 
-    public void addMarkToGrid(int position, String mark) {
+    public void takeSquare(int position, String playerMark) {
         int index = position - 1;
-        grid.setSquare(index, mark);
+        grid.setSquare(index, playerMark);
+    }
+
+    public boolean moveValid
+            (int position, String player1Mark, String player2Mark) {
+
+        return positionValid(position)
+                && squareAvailable(position, player1Mark, player2Mark);
+    }
+
+    public boolean winningLine(String playerMark) {
```

Comment:
So this method is called `winningLine`, but it actually checks if the player is a winner, not if a particular line has the winner. I think this could be renamed. 

I like how this method is universal for all board sizes! Good job!

Reply:
I'll rename this. Thanks!